app/models/question.rb

```diff
@@ -88,4 +92,43 @@ def sort_answers
       b.sort_value <=> a.sort_value
     end
   end
+
+  def create_action_verb
+    "Asked a Question"
+  end
+
+  def update_action_verb
+    "Updated his/her Question"
+  end
+
+  def tracking_information
+    [:id, :title, :content, :created_since]
+  end
+
```

Comment:
Yeah, that was considered, however it the thought was discarded because then we would have needed a builder in each model to construct the message, as messages don't follow the same pattern, unless we would be doing model checks.
```ruby
    case activity_type
    when 'create', 'index'
      if trackable_type == 'Question'

      elsif trackable_type == 'Answer'

      elsif trackable_type == 'Comment'

      end
    when 'update'
       #repeat if/elsif/else here again
    end
```
Unless we just want to create an interface 💭